annotate.el

```diff
@@ -2482,8 +2482,7 @@ sophisticated way than plain text."
                                       (separator-re    "\^L?\^_\^L?\^J")
                                       (has-separator-p (string-match separator-re file-contents))
                                       (has-node-p      (string-match "Node:" file-contents)))
-                                 (if (or (annotate-info-root-dir-p filename)
-                                         (and has-separator-p
+                                 (if (or (and has-separator-p
```

Comment:
The logic of `((has-sep && has-node) || (has-sep && has-info))` could be compacted to `(has-sep && (has-node || has-info))`.

Reply:
Thank @bastibe !

Your help is precious as usual!

Bye!
C.